pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerMeter.java

```diff
@@ -48,6 +48,7 @@ public enum ServerMeter implements AbstractMetrics.Meter {
   PARTIAL_UPSERT_OUT_OF_ORDER("rows", false),
   PARTIAL_UPSERT_KEYS_NOT_REPLACED("rows", false),
   UPSERT_OUT_OF_ORDER("rows", false),
+  DELETED_KEYS_TTL_PRIMARY_KEY_REMOVED("rows", false),
```

Comment:
+1 on adding this meter. Shall we keep removed keys from metadata TTL and delete TTL separately?

Reply:
Added 2 meters.